python/trifinger_simulation/collision_objects.py

```diff
@@ -106,42 +103,60 @@ def __init__(
             lateralFriction=lateral_friction,
             spinningFriction=spinning_friction,
             restitution=restitution,
-            **self._kwargs,
+            physicsClientId=self._pybullet_client_id,
         )
 
     def set_state(self, position, orientation):
         """
-        Resets the block state to the provided position and
-        orientation
+        Resets the cuboid to the provided position and orientation
 
         Args:
-            position: the position to which the block is to be
-                set
-            orientation: desired to be set
+            position: New position.
+            orientation: New orientation.
         """
         pybullet.resetBasePositionAndOrientation(
             self.block,
             position,
             orientation,
-            **self._kwargs,
+            physicsClientId=self._pybullet_client_id,
         )
 
     def get_state(self):
         """
         Returns:
-            Current position and orientation of the block.
+            Current position and orientation of the cuboid.
         """
         position, orientation = pybullet.getBasePositionAndOrientation(
             self.block,
-            **self._kwargs,
+            physicsClientId=self._pybullet_client_id,
         )
```

Comment:
I don't remember exactly why we are returning lists here, but then usually wherever this is called, these entities are further converted to arrays, so maybe we could already return them as arrays from here.